server-common/src/main/java/org/apache/kafka/server/group/share/DeleteShareGroupStateRequestDTO.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.server.group.share;
+
+import org.apache.kafka.common.Uuid;
+import org.apache.kafka.common.message.DeleteShareGroupStateRequestData;
+
+public class DeleteShareGroupStateRequestDTO implements PersisterDTO {
```

Comment:
I don't think we use "DTO" anywhere else in Kafka. I can see why just using `DeleteShareGroupStateRequest` and `DeleteShareGroupStateResponse` could be confusing because they have the same names as the classes generated from the RPC definitions. Personally, I could tolerate having the same class names in two packages. Alternatively, maybe you could use `DeleteShareGroupStateParameters` and `DeleteShareGroupStateResult`.

Reply:
Makes sense